lib/commands/install/index.ts

```diff
@@ -0,0 +1,129 @@
+import { AbstractCommand } from "../abstract-command";
+import { spawn } from "child_process";
+import path from "path";
+import optionModel from "../option-model.json";
+import fs from "fs";
+import CliError from "../../exceptions/cli-error";
+import CliFileNotFoundError from "../../exceptions/cli-file-not-found-error";
+import CliWarn from "../../exceptions/cli-warn";
+import { OptionModel } from "../option-validator";
+import Messenger from "../../view/messenger";
+import SpinnerView, { TERMINATE_STYLE } from "../../view/spinner-view";
+import ResourcesConfig from "../../model/resources-config";
+import CONSTANTS from "../../utils/constants";
+import stringUtils from "../../utils/string-utils";
+import profileHelper from "../../utils/profile-helper";
+
+export default class InstallCommand extends AbstractCommand {
+  name() {
+    return "install";
+  }
+
+  description() {
+    return "install alexa skill components in your alexa skill";
+  }
+
+  requiredOptions() {
+    return ["component-name"];
+  }
+
+  optionalOptions() {
+    return ["profile"];
+  }
+
+  async handle(cmd: Record<string, any>): Promise<void> {
+    let profile: string;
+    try {
+      profile = profileHelper.runtimeProfile(cmd.profile);
+      new ResourcesConfig(path.join(process.cwd(), CONSTANTS.FILE_PATH.ASK_RESOURCES_JSON_CONFIG));
+
+    } catch (err) {
+      if (err instanceof CliWarn) {
+        Messenger.getInstance().warn(err.message);
+      } else if (err instanceof CliFileNotFoundError) {
+        Messenger.getInstance().warn(err.message);
+      } else if (err instanceof CliError) {
+        Messenger.getInstance().error(err.message);
+      } else {
+        Messenger.getInstance().error(err);
+      }
+      throw err;
+    }
+
+    let skillPackageSrc = ResourcesConfig.getInstance().getSkillMetaSrc(profile);
+    if (!stringUtils.isNonBlankString(skillPackageSrc)) {
+      Messenger.getInstance().error("Skill package src is not found in ask-resources.json.");
+      return;
+    }
+    if (!fs.existsSync(skillPackageSrc)) {
+      Messenger.getInstance().error(`File ${skillPackageSrc} does not exist.`);
+      return;
+    }
+
+    const componentName = cmd.componentName;
+    const cwd = process.cwd();
+
+    // Check if the component is already installed
+    const componentDirPath = path.join(
+      skillPackageSrc,
+      "skillComponents",
+      componentName
+    );
+    const configFilePath = path.join(componentDirPath, "README.md");
+
+    if (fs.existsSync(configFilePath)) {
+      Messenger.getInstance().info(
+        `Component ${componentName} is already installed. Please use the "ask update" command to update to the latest version.`
+      );
+      return;
+    }
+
+    // Install the component using npm with spinner
+    const spinner = new SpinnerView({ color: "yellow" });
+    spinner.start(`Installing ${componentName}...`);
+
+    const installCommand = spawn("npm", ["install", componentName], {
```

Comment:
Is this PR doing an npm install on the --component-name ? If so, I failed to see the target audience that needs help with npm install.

Reply:
It's also copying the config.jsonc file to the skillPackage. This config.jsonc file needs to be configured(adding intents, for example) by the skill developer to use the component.